.github/workflows/sandbox-build-push.yml

```diff
@@ -56,6 +56,18 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: Get image tag
```

Comment:
I'd suggest expanding this to clarify which _image_; is this a base image during building, a final image tag to be published to ECR, or an image version _required_ by a given sandbox environment?

Reply:
comments added in the script